.github/workflows/main.yml

```diff
@@ -8,8 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         # Tested versions based on dates in https://devguide.python.org/devcycle/#end-of-life-branches
-        # 3.10 not yet supported, see https://github.com/octodns/octodns-ovh/issues/2
-        python-version: ['3.7', '3.8', '3.9']
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
Heh. Was in the process of making a comment about this when I saw the commit come in. There's a 3.9 down below that can be bumped to 3.10 as well at which point this should be gtg.